bHY6yh-d35397/cod-bxxqqq_bHY6yh-d35397.xml

```diff
@@ -0,0 +1,130 @@
+               </p>
+            </availability>
+         </publicationStmt>
+         <sourceDesc>
+            <listWit>
+              <witness xml:id="B" n="cod-bxxqqq">Paris, Bibliothèque Nationale de France, Nouv. acq. lat. 2469</witness>
+            </listWit>
+         </sourceDesc>
+      </fileDesc>
+      <encodingDesc>
+         <schemaRef n="lbp-diplomatic-1.0.0" url="https://raw.githubusercontent.com/lombardpress/lombardpress-schema/1.0.0/src/out/diplomatic.rng"/>
+         <editorialDecl>
+            <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+       guidelines for a diplomatic edition.</p>
+         </editorialDecl>
+      </encodingDesc>
+      <revisionDesc status="draft">
+         <listChange>
+            <change when="2023-08-12" status="draft" n="0.0.0">
+               <p>Created file for the first time.</p>
+            </change>
+         </listChange>
+      </revisionDesc>
+   </teiHeader>
+   <text xml:lang="la">
+      <front>
+         <div xml:id="starts-on">
+            <pb ed="#B" n="58-r"/>
+            <cb ed="#B" n="a"/>
+         </div>
+      </front>
+      <body>
+         <div xml:id="bHY6yh-d35397"><!-- p2l4-r14 -->
+            <head xml:id="bHY6yh-d35397-Hd1e108">Rubrica 14</head>
+            <head xml:id="bHY6yh-d35397-Hd1e111" type="question-title">De causis inducentibus pacem et concordiam</head>
+            <p xml:id="bHY6yh-d35397-d1e114">
+              <lb ed="#B"/>De causis inducentibus pacem<!--line jumps down--> et concordiam Rubrica 14
+            </p>
+            <p xml:id="bHY6yh-d35397-d1e121">
+              <lb ed="#B"/><ref xml:id="bHY6yh-d35397-Rd1e117">ARistotiles polit<lb ed="#B" break="no"/>icorum 
+              libro 5 capitulo 3</ref> <quote xml:id="bHY6yh-d35397-Qd1e129">Congregat 
+              <lb ed="#B"/>etiam separatissimos communis timor</quote>
+            </p>
+            <p xml:id="bHY6yh-d35397-d1e135">
+              <cb ed="#B" n="b"/>
+              <lb ed="#B"/><ref xml:id="bHY6yh-d35397-Rd1e134">Lvcas in ewangelio capitulo 23 et penultimo</ref> <quote xml:id="bHY6yh-d35397-Qd1e139">Mul<lb ed="#B" break="no"/>titudo
+              iudeorum duxerunt ihesum 
+              <lb ed="#B"/>ad pilatum etc dicens commouet po<lb ed="#B" break="no"/>pulum 
+              dicens per vniuersam iudeam 
+              <lb ed="#B"/>et incipiens a galilea usque hic Pi<lb ed="#B" break="no"/>latus 
+              autem audiens galileam: in<lb ed="#B" break="no"/>terrogauit
+              si homo galileus esset 
+              <lb ed="#B"/>et ut cognouit quod de herodis potesta<lb ed="#B" break="no"/>te 
+              esset remisit eum ad herodem
+              <lb ed="#B"/>qui et ipse yerosolimis erat illis 
+              <lb ed="#B"/>diebus & herodes autem viso ihesu 
```

Comment:
did you mean & here and throughout? or did you mean to use a pilcrow. 

& is a special character in XML, and therefore to write &, you have to write `&amp;`. 

But I haven't seen your manuscript use this character. Please double check.